integration_test/gce/gce_testing.go

```diff
@@ -1731,3 +1733,33 @@ func RunForEachPlatform(t *testing.T, f func(t *testing.T, platform string)) {
 func ArbitraryPlatform() string {
 	return strings.Split(os.Getenv("PLATFORMS"), ",")[0]
 }
+
+func AddTagToVm(ctx context.Context, logger *log.Logger, vm *VM, tag string) (CommandOutput, error) {
```

Comment:
Since the underlying command `add-tags` supports multiple tags, what do you think about making the `tag` argument a []string? We could always make that change later, but I think there's a certain appeal to making this function map onto `add-tags` directly, without adding new restrictions like "only 1 tag at a time".

optional suggestion.